openapi/components/schemas/Subscription.yaml

```
@@ -339,7 +339,10 @@ properties:
       - 'null'
     description: |-
       Length of time, in ISO-8601 durations format,
-      before which an order with an invoice in past the due state is automatically canceled.
+      which is added to the due time when setting the delinquency time of every order's invoice.
+      Reaching delinquency time of an invoice makes the order automatically canceled,
+      which abandons that and other order's invoices with a status `unpaid` or `past-due`.
```

nitpick: [possessive apostrophes](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#possessive-apostrophes)
```suggestion
      which is added to the due time of the order when setting the delinquency time for all related invoices.
      When the delinquency time of an invoice is reached, the order is automatically canceled. 
      The current invoice is abandoned along with order invoices that have the `unpaid` or `past-due` status.
```